Mlem/API/Models/Site/APILocalSite.swift

```diff
@@ -38,7 +38,7 @@ struct APILocalSite: Decodable {
 
 // lemmy_db_schema::source::local_site::RegistrationMode
 enum APIRegistrationMode: String, Codable {
-    case closed = "Closed"
-    case requireApplication = "RequireApplication"
-    case open = "Open"
+    case closed = "closed"
+    case requireApplication = "requireapplication"
+    case open = "open"
```

Comment:
Hey @ShadowJonathan - I had to make a couple of changes in this model as it was unable to decode, the first one `actorNameMaxLength` I can verify against the Rust side, but for this string based enum I was struggling to confirm why it was all lowercase in my responses (this was on `beehaw` and `test-mlem`) - are you able to comment on whether this is expected across all instances, or do we need to handle both `closed` and `Closed`?

Reply:
`#[serde(rename_all = "lowercase")]`

should be fine as-is, thanks for checking, and sorry for my tardiness